src/transforms.jl

```diff
@@ -40,15 +40,42 @@ clrinv(x) = Composition(𝓒(exp.(x)))
 
 Isometric log-ratio transformation of composition `c`.
 """
-function ilr(c::Composition)
-  # TODO
+function ilr(c::Composition{D,SYMS}) where {D,SYMS}
+  log_parts = log.(c.parts .+ eps())
+  x = @MVector zeros(D-1)
+  for i in 1:D-1
+    s = 0.
```

Comment:
I think it is fine to initially with a Float64 given that the implementation is currently limited to this type.